.eslintrc.js

```diff
@@ -5,8 +5,15 @@ module.exports = {
     es6: true,
   },
   parserOptions: { ecmaVersion: 8 }, // to enable features such as async/await
-  ignorePatterns: ['node_modules/*', '.next/*', '.out/*', '!.prettierrc.js', ], // We don't want to lint generated files nor node_modules, but we want to lint .prettierrc.js (ignored by default by eslint)
-  extends: ['eslint:recommended', 'plugin:react/recommended'],
+  // We don't want to lint generated files nor node_modules, but we want to lint .prettierrc.js (ignored by default by eslint)
+  ignorePatterns: ['node_modules/*', '.next/*', '.out/*', '!.prettierrc.js'],
+  extends: [
+    'eslint:recommended',
+    'next',
+    'plugin:yml/standard',
+    'plugin:yml/prettier',
+    'prettier',
```

Comment:
Since eslint isn't used for formatting, nor does it acctually show any errors, why have these in the eslint config?

Reply:
It disables rules in eslint that conflict with prettier https://prettier.io/docs/en/integrating-with-linters.html

Reply:
oh great cheers